Rakefile

```diff
@@ -20,8 +20,8 @@ task :test do
 			"texthighlight/texthighlight-fr.html?txthl=influenza%20aviaire+monde+suffis+symptômes%20semblables%20à%20ceux%20de%20l'influenza+À%20titre%20de%20rappel...+À%20de%20rares%20occasions,%20des%20humains%20ont%20été%20infectés%20par%20ce%20virus.",
 			"../../../demos/texthighlight/texthighlight-en.html?txthl=influenza%20aviaire+monde+suffis+sympt%C3%B4mes%20semblables%20%C3%A0%20ceux%20de%20l'influenza+%C3%80%20titre%20de%20rappel...+%C3%80%20de%20rares%20occasions,%20des%20humains%20ont%20%C3%A9t%C3%A9%20infect%C3%A9s%20par%20ce%20virus.#exemple"
 		],
+		:allow_missing_href => true,
 		:disable_external => true,
-		:empty_alt_ignore => true,
-		:ext => ".html"
+		:enforce_https => false
```

Comment:
@EricDunsworth Would be great to eventually convert our http links to https / replace them by others instead of ignoring them. I'll create an issue so we track it but please @EricDunsworth feel free to open it if you feel like it :) Pretty sure you'll be quicker than me.

Reply:
@GormFrank : Eric already did that change a long time ago. Here, I do think it is because this test are running on a local http server.